lib/main.dart

```diff
@@ -30,12 +31,17 @@ class MyApp extends StatelessWidget {
   Widget build(BuildContext context) {
     return MaterialApp(
       debugShowCheckedModeBanner: false,
-      home: MultiBlocProvider(providers: [
-        BlocProvider(create: (context) => _di.get<CoursesCubit>()),
-        BlocProvider(create: (context) => _di.get<AuthCubit>()),
-        BlocProvider(create: (context) => _di.get<TheoryCubit>()),
-        BlocProvider(create: (context) => _di.get<ProfileCubit>()),
-      ], child: const MyHomePage()),
+      initialRoute: AppRouting.home,
+      onGenerateRoute: AppRouting.generateRoute,
+      home: MultiBlocProvider(
+        providers: [
+          BlocProvider(create: (context) => _di.get<CoursesCubit>()),
+          BlocProvider(create: (context) => _di.get<AuthCubit>()),
+          BlocProvider(create: (context) => _di.get<TheoryCubit>()),
+          BlocProvider(create: (context) => _di.get<ProfileCubit>()),
```

Comment:
should remove from here as you provide it in app routing

Reply:
ok